gcovr/formats/html/write.py

```diff
@@ -85,6 +84,7 @@ def templates(options):
                 PackageLoader("gcovr.formats.html"),
             ]
         )
+        print("IN LOADER")
```

Comment:
This print must be removed.

Reply:
Oh jeez, that's embarrassing! Was rushing to get it in before I had to catch a flight.

Reply:
No problem.